policy-test/tests/e2e_authorization_policy.rs

```diff
@@ -48,6 +50,132 @@ async fn meshtls() {
     .await;
 }
 
+#[tokio::test(flavor = "current_thread")]
+async fn targets_route() {
+    with_temp_ns(|client, ns| async move {
+        // First create all of the policies we'll need so that the nginx pod
```

Comment:
the nitiest of nits: think we replaced nginx with [hokay](https://github.com/olix0r/hokay). Is that the case here too? Would it be worth updating the comment to replace nginx -> web?

Reply:
```suggestion
        // First create all of the policies we'll need so that the web pod
```